examples/Replay

```diff
@@ -0,0 +1,46 @@
+#include <Servo.h>
+#include <EEPROM.h>
+
+Servo myServo;        // create servo object to control a servo
+
+bool record = false;  // boolean to determine if recording is taking plce
+int potpin = 0;       // analog pin used to connect the potentiometer
```

Comment:
```suggestion
int potPin = 0;       // analog pin used to connect the potentiometer
```
The established variable name style for official Arduino example sketches is camelCase.